internal/database/models/models.go

```diff
@@ -17,12 +18,14 @@ type Author struct {
 	Stories       []Story        `gorm:"foreignKey:AuthorFirstName,AuthorLastName,AuthorCountry;references:FirstName,LastName,OriginCountry;constraint:OnUpdate:CASCADE,OnDelete:CASCADE;" json:"stories,omitempty"`
 }
 
+// Story struct describes a story rendered on the map
 type Story struct {
 	gorm.Model
 	Title           string  `gorm:"not null" json:"title"`
 	Content         string  `gorm:"type:text;not null" json:"content"`
 	CurrentCity     string  `gorm:"not null" json:"current_city"`
 	Year            uint    `gorm:"not null" json:"year"`
+	IsPublic        bool    `gorm:"not null" json:"is_public"`
```

Comment:
I think we can set a default value as false `gorm:"default:false"`, rather than setting story.isPublic to False later on